dev-docs/modules/floors.md

```diff
@@ -474,6 +481,77 @@ Additionally skipRate is supplied at model group level where model1 will skip fl
 }
 {% endhighlight %}
 
+## Custom Schema Fields
+
+Custom schema fields are fields the Floors Module does support out of the box. To use a custom schema field, one needs to perform twp steps:
+
+1. Create lookup function to give the Floors Module context of the value of custom fields for that given auction
+1. Define, Set and Map Custom Schema Attributes
+
+### Create Lookup Function
+
+Create a function to allow the Floors Module to understand context of a given auction. In the below example, we must create a lookup function to give the Floors Module what deviceType this auction is. 
+
+Here is an example lookup function:
+
+{% highlight js %}
+
+  function deviceTypes (bidRequest, bidResponse) {
+      //while bidRequest and bidResponse are not required for this function, they are available for custom attribute mapping
+      
+      let deviceType = getDeviceTypeFromUserAgent(navigator.userAgent);
+
+      if(deviceType = 'mobile')
+          return 'mobile'
+      else if (deviceType = 'tablet')
+          return 'tablet'
+      else if (deviceType = 'desktop')
+          return 'desktop'
+  }
+
+{% endhighlight %}
+
+
+### Define, Set and Map Custom Schema Attributes
+
+After defining a lookup function for the given context of the auction, the custom schema field(s) need to be defined in the `floors.schema.fields` array. Once your custom field is defined you can assign rule values in `floors.data.value` derived from said field(s). The last step would be to supply the lookup function(s) that map from each custom field to a value of the context wthin that auction by using the `floors.additionalSchemaFields` attribute as seen below.
```

Comment:
should `floors.data.value`

be

`floors.data.values`

Reply:
corrected